add-on/panels/css/panel.css

```diff
@@ -96,6 +96,67 @@ body {
   border-radius: 3px;
 }
 
+/* Rename closing room panel */
+.rename-newRoom > img {
+  margin-left: 16px;
+  position: absolute;
+  width: 32px;
+}
+html[dir="rtl"] .rename-newRoom > img {
+  margin-right: 16px;
+}
+
+.rename-container {
+  margin: 16px;
+  /* margin is img-width + 16px separation */
+  padding-left: 48px;
+}
+html[dir="rtl"] .rename-container {
+  padding-right: 48px;
```

Comment:
Similarly, add a `padding-left: 0px;` here to reset the left padding, otherwise RTL locales will have less width.